cves/CVE-2017-9793.yml

```diff
@@ -225,3 +231,9 @@ mistakes:
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
   answer:
+    This vulnerability was exploited due to an outdated library dependency that
+    the REST Plugin on this application utilizes. The ideal scenario to avoid this sort
+    of vulnerability is to restrict the use of third party dependencies, this can be mitigated
+    by either redesigning the architecture of the plugin or attempting to design an internal library
+    that was thorougly tested before production. It is always important to be wary of third-party
+    dependencies which are constantly targeted by attackers, especially for open source software like Struts.
```

Comment:
Do you think trust boundaries could have helped mitigate this issue?

Reply:
Yes indeed, if trust boundaries were implemented where the application would validate input that affects critical system functions, and deny anything that is not trustful this type of vulnerability could have been avoided.